2.process.py

```diff
@@ -40,6 +42,7 @@ def create_gene_df(path):
     renamer = collections.OrderedDict([
         ('GeneID', 'entrez_gene_id'),
         ('Symbol', 'symbol'),
+        ('dbXrefs', 'other_id'),
```

Comment:
Here's an example of what's in this column:

```
MIM:601065|HGNC:HGNC:20|Ensembl:ENSG00000090861|Vega:OTTHUMG00000177042
```

Since this field is a list, the column should be plural... `other_ids`.